.circleci/config.yml

```diff
@@ -20,6 +20,21 @@ jobs:
           covermode: atomic
           failfast: true
           race: true
+  publish-bot:
+    environment:
+      NODE_AUTH_TOKEN: $NPM_TOKEN  # Use NPM_TOKEN as the auth token
+    docker:
+      - image: cimg/node:18  # Use Node.js 18
+    working_directory: ~/superchain-registry/
+    steps:
+      - checkout  # Checkout the code
+      - run:
+          name: Setup .npmrc for npm publishing
```

Comment:
Is the `.npmrpc` optional? cc @roninjin10 would appreciate some pointers for NPM publishing best-practices

Reply:
I think you can use env variables but nearly everywhere I see uses an npmrc like here